runtime/doc/eval.txt

```diff
@@ -4704,6 +4706,36 @@ getmatches([{win}])					*getmatches()*
 			'pattern': 'FIXME', 'priority': 10, 'id': 2}] >
 			:unlet m
 <
+getmousepos()						*getmousepos()*
+		Returns a Dictionary with the last known position of the
+		mouse.  This can be used in a mapping for a mouse click.  The
+		items are:
+			screenrow	screen row
+			screencol	screen column
+			winid		Window ID of the click
+			winrow		row inside "winid"
+			wincol		column inside "winid"
+			line		text line inside "winid"
+			column		text column inside "winid"
+		All numbers are 1-based.
+
+		If not over a window, e.g. when in the command line, then only
+		"screenrow" and "screencol" are valid, the others are zero.
+
+		When on the status line below a window or the vertical
+		separater right of a window, the "line" and "column" values
+		are zero.
+
+		When the position is after the text then "column" is the
+		length of the text in bytes.
```

Comment:
This documentation was ported directly from Vim. The documentation currently does not match the behavior, since `getmousepos().column` in this case would be "the length of the text in bytes" _plus one_. Vim's help will be [updated](https://github.com/vim/vim/issues/8474#issuecomment-869953539) to correct this. I haven't made any updates here, with the intent that the future update to the Vim documentation be ported to Neovim, with the text kept in sync.

Reply:
The documentation was corrected in Vim as part of https://github.com/vim/vim/commit/90df4b9d423485f7db16e3a65cab4f14edc815ae. I've applied this same update in 3dbf80e250bde2edc8587a8c75073b062547c415.